src/controllers/scenario/index.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Controller,
+  Get,
+  Path,
+  Route,
+} from "tsoa";
+import config from 'config'
+import { exec } from 'child_process'
+// import { Scenario } from '../../../types'
+import Logger from '../../utils/Logger'
+
+let log = Logger.child({ controller: '/scenario', morello_host: config.get('morello_host') });
+const { address, user } = config.get('morello_host')
+
+@Route('scenario')
+export class scenario extends Controller {
+  @Get('{id}')
+  public async get(@Path() id: string): Promise<any> {
```

Comment:
I'm not sure I understand how this controller works. Why is a get controller doing an`scp`?

Reply:
Mmm.... to be faiir I'm a little confused in ergards to scp, this is something I would like too have a script rather than api doing, lets talk after or during out 1-2-1

Reply:
Mmm... going to mark this as done as we have already discussed